e2e/app.e2e.ts

```diff
@@ -1,5 +1,6 @@
 const Application = require('spectron').Application;
 const { resolve } = require('path');
+const clone = require('clone');
```

Comment:
do we need this extra dependency? we are trying to limit number of dependencies in package.json to minimum, but if you find out that we need this then also install `@types/clone` and import this as `import * as clone from 'clone';`. that's the typescript way.